bundlers/silius/silius.yml

```diff
@@ -20,6 +20,8 @@ services:
       - "3000"
       - --http.api
       - eth,debug,web3
+      - --poll-interval
+      - "5"
```

Comment:
Are you sure you need it? Afaik, the tests are all manual, never let a timeout to create a bundle

Reply:
Yes, it's needed. This is not related to bundle creation, but mempool sync and included user operations on-chain